src/Components/EndGameScreen.css

```diff
@@ -0,0 +1,53 @@
+button{
+    border-radius: 5px;
+}
+
+.end-game-container{
+    display: flex;
+    position: absolute;
```

Comment:
I needed this so that the div overlaps the end game screen. If I remove it, then the whole screen would start 1 vh after where it's meant to. I did however remove it from the `.end-game` one